azure-storage-cpp-lite/include/retry.h

```diff
@@ -67,8 +68,11 @@ namespace microsoft_azure {
             retry_info evaluate(const retry_context &context) const override {
                 if (context.numbers() == 0) {
                     return retry_info(true, std::chrono::seconds(0));
-                } else if (context.numbers() < 100 && can_retry(context.result())) {
-                    return retry_info(true, std::chrono::seconds(5+rand()%5));
+                } else if (context.numbers() < 60 && can_retry(context.result())) {
+                    double delay = (pow(2, context.numbers()-1)-1);
+                    delay *= (((double)rand())/RAND_MAX)/2 + 0.8;
```

Comment:
Minor - this should probably be +0.75 :)  +0.8 puts the jitter in [0.8, 1.3], which biases above 1 (again though, not that it really matters here :) )

Reply:
I think 0.75 will be easier to reason about than 0.8. And it would be nice to give them names